core/files/css/base.php

```diff
@@ -340,6 +340,13 @@ public function add_control_rules( array $control, array $controls_stack, callab
 				}
 			} else {
 				try {
+					if ( isset( $control['unit_selectors_dictionary'] ) && isset( $control['unit_selectors_dictionary'][ $values[ $control['name'] ]['unit'] ] ) ) {
+						$css_property = $this->parse_size_units_selectors_dictionary(
```

Comment:
Same question as I raised above in the JS parser - do we need to even parse the selectors here? isn't it done below in the `preg_replace_callback()` call? To my understanding, all that needs to be done here, is to check if the `unit_selectors_dictionary` has an entry for the control's unit, and if it does, replace the `$css_property` variable value with the dictionary entry value, no?